utilities/test_suite/HOST/Tensor_host.cpp

```diff
@@ -556,7 +556,7 @@ int main(int argc, char **argv)
                 startWallTime = omp_get_wtime();
                 startCpuTime = clock();
                 if (inputBitDepth == 0 || inputBitDepth == 1 || inputBitDepth == 2 || inputBitDepth == 3 || inputBitDepth == 4 || inputBitDepth == 5)
-                    rppt_crop_mirror_normalize_host(input, srcDescPtr, output, dstDescPtr, offset, multiplier, mirror, roiTensorPtrSrc, roiTypeSrc, handle);
+                    rppt_crop_mirror_normalize_host(input, srcDescPtr, output, dstDescPtr, offset, multiplier, mirror, roiTensorPtrDst, roiTypeSrc, handle);
```

Comment:
All other geometric functions ideally take roiTensorPtrSrc. But if this is destination roi, please change the param description comment in the rpp/include .h file too to explain what roi is to be provided.

Reply:
Hi Abishek,
for CMN we will give the destination crop co-ordinates in roiTensorPtrSrc

Here in this case we are updating roiTensorPtrSrc like below, Incase of performance tests this is giving issues when when update roiTensorPtrSrc for every iteration
`for (i = 0; i < images; i++)
                {
                    roiTensorPtrDst[i].xywhROI.xy.x = 10;
                    roiTensorPtrDst[i].xywhROI.xy.y = 10;
                    dstImgSizes[i].width = roiTensorPtrDst[i].xywhROI.roiWidth = roiTensorPtrSrc[i].xywhROI.roiWidth / 2;
                    dstImgSizes[i].height = roiTensorPtrDst[i].xywhROI.roiHeight = roiTensorPtrSrc[i].xywhROI.roiHeight / 2;
                }`

I am not sure if it is necessary to add a comment for this

Reply:
@sampath1117 Okay, as long as there is clarity on what the user needs to supply in the documentation, its fine